sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModelInfo.cs

```diff
@@ -18,18 +20,22 @@ internal CustomFormModelInfo(ModelInfo_internal modelInfo)
         }
 
         /// <summary>
+        /// The identifier of the model.
         /// </summary>
         public string ModelId { get; }
 
         /// <summary>
+        /// The status of the model.
         /// </summary>
         public CustomFormModelStatus Status { get; }
 
         /// <summary>
+        /// The date and time (UTC) when the model was created.
```

Comment:
Let's use the same descriptions for these as in CustomFormModel -- these are exactly the same fields.
```
        /// The date and time (UTC) when model training was started.
        /// </summary>	        /// </summary>
        public DateTimeOffset CreatedOn { get; }	        public DateTimeOffset CreatedOn { get; }


        /// <summary>	        /// <summary>
        /// Date and time (UTC) when model training completed.	        /// The date and time (UTC) when model training completed.
        /// </summary>	        /// </summary>
        public DateTimeOffset LastModified { get; }	        public DateTimeOffset LastModified { get; }
```